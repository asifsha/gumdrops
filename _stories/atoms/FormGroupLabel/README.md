The `<FormGroupLabel>` component is used to add a label to your `<FormGroup>`.

**Example:**

```
<FormGroup>
    <FormGroupLabel text="Password"/>
    <TextInput  ... />
</FormGroup>
```
