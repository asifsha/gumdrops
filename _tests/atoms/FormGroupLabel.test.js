/* globals mount */
import React from 'react';
import FormGroupLabel from '../../components/atoms/FormGroupLabel';

const defaultProps = {
    text: 'Label Baby Jr',
    className: 'my-classy-class'
};

describe('Expect <FormGroupLabel>', () => {
    it('to render', () => {
        const wrapper = mount(<FormGroupLabel {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });
});
