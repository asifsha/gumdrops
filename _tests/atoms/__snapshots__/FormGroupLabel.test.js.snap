// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <FormGroupLabel> to render 1`] = `
<FormGroupLabel
  className="my-classy-class"
  text="Label Baby Jr"
>
  <label
    className="gds-form-group__label my-classy-class"
  >
    Label Baby Jr
  </label>
</FormGroupLabel>
`;
